### PR TITLE
Format `CITATION.cff` as a `.cff` file

### LINKED ISSUE #517: The citation file cannot be parsed
When clicking the "cite this repository" button in the about panel on the main page of our repository, I get the message

> Your CITATION.cff file cannot be parsed. Make sure the formatting is correct.

### PR DESCRIPTION
**Pull request by fredrik-bakke**
- Add me as co-author
- Fix formatting of citation file
- Remove citation information in GitHub readme, as it is already available under the "Cite this repository" button

Resolves #517.

### CODE REVIEW

**Comment by EgbertRijke** [comment]
I suggest that you add your name to the list too

**Comment by fredrik-bakke** [comment]
This is GitHub's BibTeX output:

```bib
@software{Rijke_Univalent_mathematics_in,
author = {Rijke, Egbert and Bonnevier, Elisabeth and Prieto-Cubides, Jonathan and {others}},
title = {{Univalent mathematics in Agda}},
url = {https://github.com/UniMath/agda-unimath/}
}
```

**Comment by fredrik-bakke** [comment]
> I suggest that you add your name to the list too

Oh, thank you for the offer, but I'm unsure about this. I'm clearly not a cofounder of this project.

**Comment by EgbertRijke** [comment]
> > I suggest that you add your name to the list too
> 
> Oh, thank you for the offer, but I'm unsure about this. I'm clearly not a cofounder of this project.

You're on the team of maintainers now:)

**Comment by fredrik-bakke** [comment]
Alright, if @elisabethbonnevier and @jonaprieto are okay with this, then I would be honored! 😊

File: CITATION.cff [hunk 1]
```diff
@@ -1,6 +1,21 @@
-@misc{Agda-UniMath,
-  author =     {Egbert Rijke and Elisabeth Bonnevier and Jonathan Prieto-Cubides and others},
-  title =     {Univalent mathematics in {Agda}},
-  url =         {https://github.com/UniMath/agda-unimath/},
-  howpublished = {\url{https://unimath.github.io/agda-unimath/}}
-}
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+title: "Univalent mathematics in Agda"
+abstract: "The agda-unimath library is a community formalisation project for univalent mathematics in Agda. Our goal is to formalize an extensive curriculum of mathematics from the univalent point of view. Furthermore, we think libraries of formalized mathematics have the potential to be useful, and informative resources for mathematicians. Our library is designed to work towards this goal, and we welcome contributions to the library about any topic in mathematics."
```
Comment (jonaprieto) [inline CITATION.cff:4]: Hehe, I was too far from the correct format here.
Thanks!

**Review by jonaprieto — APPROVED** [review]
:)

**Pull request merged**